<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QMainWindow" name="credit">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2f3030;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2f3030;
color:white</string>
    </property>
    <property name="text">
     <string>Ежемесячный платеж:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background: qlineargradient(x1:0, y1:0, x2:3 y2:0,
                                   stop:0 #4671D5, stop: 0.3 #F56E8D, stop:1 #CF5FD3);
border-radius: 28px
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);

}
</string>
    </property>
    <property name="text">
     <string>Рассчитать:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2f3030;
color:white</string>
    </property>
    <property name="text">
     <string>Переплата по кредиту:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2f3030;
color:white</string>
    </property>
    <property name="text">
     <string>Общая выплата:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2f3030;
color:white
</string>
    </property>
    <property name="text">
     <string>Calculation results:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #adadad;
color:black;
border-radius:30px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>Credit calculator:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sum_credit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #A4A3A1;
  border: 1px solid gray;
border-radius: 5px;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>Сумма кредита:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>₽</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="percent_credit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #A4A3A1;
  border: 1px solid gray;
border-radius: 5px;
color:black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>Процентная ставка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>Срок кредита:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>280</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>лет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="term_credit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #A4A3A1;
  border: 1px solid gray;
border-radius: 5px;
color:black;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #A4A3A1;
border: 1px solid gray;
color:black;
border-radius: 5px</string>
    </property>
    <item>
     <property name="text">
      <string>Аннуитетный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дифференцированный</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B1B0AE;
color:black
</string>
    </property>
    <property name="text">
     <string>Порядок погашения:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> white-space: normal;
background-color: #A4A3A1;
color: white;
  border: 1px solid gray;
border-radius: 5px;


</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> white-space: normal;
background-color: #A4A3A1;
color: white;
  border: 1px solid gray;
border-radius: 5px;


</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> white-space: normal;
background-color: #A4A3A1;
color: white;
  border: 1px solid gray;
border-radius: 5px;


</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
